app/views/posts/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="a-post">
+<div class="post-id ">
+  <div class="post-info">
+      <h4>Post <%= @post.id %> by <%= @post.user.name %></h4>
```

Comment:
- [ ] For the redirect after creating a post to work you need to change `user` to `author` because this throws errors.